docs/kubernetes/kctlr-multi-tenancy.rst

```diff
@@ -0,0 +1,225 @@
+.. index::
+   single: BIG-IP Controller; Kubernetes; multi-tenancy
+   single: BIG-IP Controller; Openshift; multi-tenancy
+
+.. _openshift multi-tenancy:
+
+Multi-tenancy in Kubernetes and OpenShift
+=========================================
+
+Overview
+--------
+
+BIG-IP administrative partitions allow you to isolate tenants from each other and to configure objects for specific tenants, where only authorized users can access them. To manage multi-tenant BIG-IP systems with the |kctlr-long| and OpenShift, F5 recommends deploying one (1) instance of the :code:`k8s-bigip-ctlr` per BIG-IP partition.
+
+You can set up the |kctlr| for multi-tenancy a few different ways. Each of the example use cases described here uses namespaces (or projects, in OpenShift) to achieve multi-tenancy in the Kubernetes/OpenShift Cluster.
```

Comment:
Should we link out to [Projects](https://docs.openshift.org/latest/dev_guide/projects.html#dev-guide-projects) documentation for OpenShift here? 

Most oc commands run in the context of a Project. We don't want customers to confuse the 2 as a namespace in k8s is not quite the same as a Project in OpenShift.

Thoughts @f5yacobucci ?

Reply:
Rewritten to clarify.